BUILD.gn

```diff
@@ -186,29 +211,7 @@ executable("snapshot_creator") {
 # in the runtime bundle
 run_node("gen_declarations") {
   out_dir = target_gen_dir
-  sources = [
-    "js/assets.ts",
-    "js/compiler.ts",
-    "js/console.ts",
-    "js/deno.ts",
-    "js/dispatch.ts",
-    "js/errors.ts",
-    "js/fetch.ts",
-    "js/global-eval.ts",
-    "js/globals.ts",
-    "js/mkdir.ts",
-    "js/os.ts",
-    "js/read_file.ts",
-    "js/remove.ts",
-    "js/stat.ts",
-    "js/text_encoding.ts",
-    "js/timers.ts",
-    "js/tsconfig.generated.json",
-    "js/types.ts",
-    "js/util.ts",
-    "js/v8_source_maps.ts",
-    "js/write_file.ts",
-  ]
+  sources = ts_sources
```

Comment:
Probably you want to add `+ ["js/plugins.d.ts" + "js/fetch_types.d.ts"]` here.

Reply:
Fixed.